src/Immediate.Handlers/Generators/ImmediateHandlers/ImmediateHandlersGenerator_Entrypoint.cs

```diff
@@ -88,7 +88,7 @@ bool hasDi
 		cancellationToken.ThrowIfCancellationRequested();
 		var source = template.Render(new
 		{
-			Handlers = handlers,
+			Handlers = handlers.Select(x => x.displayName),
```

Comment:
@dukesteen I just saw you fixed it the other way round, I'll revert this then, your change is better, since I missed to rename "Handlers" to "HandlerNames" or similar 👍

Reply:
i already reverted @dukesteen 's... ahah

Reply:
Peak coordination haha